webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -1,23 +1,24 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
+  <ds-button v-if="!notificationsCount" class="notifications-menu" disabled icon="bell">
+    {{ unreadNotificationsCount }}
   </ds-button>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-26T07:04:45Z" title="Thursday, September 26th 2019, 9:04:45 am +02:00">Sep 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,23 +1,35 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
-  </ds-button>
+  <div
+    v-if="totalNotifications <= 0"
```


Why do we have to add this extra `v-if` here? :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-26T07:44:20Z" title="Thursday, September 26th 2019, 9:44:20 am +02:00">Sep 26, 2019</time>_

---


there is a `v-if` because there is the additional functionality that the `NotificationsMenu`, if clicked on when there are no notifications, updates the notifications

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-26T07:54:33Z" title="Thursday, September 26th 2019, 9:54:33 am +02:00">Sep 26, 2019</time>_

---


Also, @Tirokk, I submit as proof I didn't purposely remove the controversial line of code with comments to explain to the `PR` reviewer. 
Do you remember when we discussed and I changed this line of code to be `totalNotifications === 0`??
Must not have saved the file, or some other quirk with `git`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-26T08:12:57Z" title="Thursday, September 26th 2019, 10:12:57 am +02:00">Sep 26, 2019</time>_

---


The extra `div` with this `v-if` is to switch between the disabled and enabled button and have the ability to update it on click even if disabled. Like @mattwr18 explained.